attestation-service/README.md

```diff
@@ -40,13 +40,22 @@ from the Attestation Agent inside the guest.
 
 The AS has a two-step verification process.
 
-1. Verify the format and provenance of evidence itself (i.e. check the signature of the evidence).
-2. Evaluate the claims presented by the evidence (i.e. check that measurements are what the client expects).
+1. Verify the format and provenance of evidence itself (i.e. check the signature of the evidence). This is done by [verifier drivers](#verifier-drivers).
+2. Evaluate the claims presented by the evidence (i.e. check that measurements are what the client expects). This is done by [policy engine](#policy-engine) and [RVPS](#reference-value-provider-service).
```

Comment:
Maybe it would be more clear to say `workload owner` instead of `client` here.

Reply:
Ok. This is not changed in this PR, but anyway let me fix this : )

Reply:
I guess I am thinking that client is ambiguous because it could mean the client of the AS API (or another project consuming the crate) but it could also mean the owner of the workload. I guess this could actually mean either one in this case so maybe it is fine if it is ambiguous.